### ISSUE TITLE
Version 19 nonfunctional on Gnome 41.1 (Fedora 35)

### ISSUE THREAD

**Issue by RolesG** [MartinPL/Tray-Icons-Reloaded#62]
Title pretty much says everything, the extension won't even start on gnome 41.1 with version 19. It worked previously with version 17.

**Comment by tr1p0p** [comment]
On mine (same, fedora 35 Gnome 41.1) the extension kind of work, but didn't display discord tray icon..

**Comment by martinpl** [comment]
No issues in virtual machine :man_shrugging: 
![image](https://user-images.githubusercontent.com/8329810/145463435-d20e6556-3293-4155-919c-6431d3972840.png)

**Comment by RolesG** [comment]
interesting, maybe try it in a live environment? to the best of my knowledge, you can still install (temporary) extensions and programs on a usb disk

**Comment by tr1p0p** [comment]
Hum, so, because it's been almost a week, and nobody seem to find the fact that push a update that's broke what's working well before for... no new features (?) and that 2 users from literally the gnome flagship distro report issues, i will, at least, give a workaround for people that have the same problems :

-Just download v17 on release page and put it on .local/share/gnome-shell/extensions. :warning: Gnome extension push and force update of extensions, you have to deactivate this behavior on the firefox/gnome extensions settings to keep the working version.

Hope this issues get taken seriously and fix soon, but in the mean time, got absolutely no problems with v17 so i dunno what i will change.

Thks still for this great tray icon extension ! (Because linux seem to want to be VERY weird and difficult for this precise thing that windows do greatly for decades now, but hey, we know that linux is like that sometimes)

EDIT : Because Gnome like to be a pain for dev AND user, the gnome-shell update extensions no matter what, even when they have options to toggle specially to avoid that. You have to edit medata file to put "19". When Gnome stop to be a fucking mess for all linux user ??

**Comment by RolesG** [comment]
Yeah, GNOME seems to be both anti-user and anti-developer these days. Good luck to MartinPL!

**Comment by NicoleCollard** [comment]
> Hum, so, because it's been almost a week, and nobody seem to find the fact that push a update that's broke what's working well before for... no new features (?) and that 2 users from literally the gnome flagship distro report issues, i will, at least, give a workaround for people that have the same problems :
> 
> -Just download v17 on release page and put it on .local/share/gnome-shell/extensions. warning Gnome extension push and force update of extensions, you have to deactivate this behavior on the firefox/gnome extensions settings to keep the working version.
> 
> Hope this issues get taken seriously and fix soon, but in the mean time, got absolutely no problems with v17 so i dunno what i will change.
> 
> Thks still for this great tray icon extension ! (Because linux seem to want to be VERY weird and difficult for this precise thing that windows do greatly for decades now, but hey, we know that linux is like that sometimes)
> 
> EDIT : Because Gnome like to be a pain for dev AND user, the gnome-shell update extensions no matter what, even when they have options to toggle specially to avoid that. You have to edit medata file to put "19". When Gnome stop to be a fucking mess for all linux user ??

i try'd to do this, but without success!

**Comment by vsudakov** [comment]
the same issue for me on Fedora 35 and Gnome 41.3.
Only version 17 of the extension works.

**Comment by stablestud** [comment]
I can confirm the issue on Fedora and Gnome 41.3 as it stopped working for me aswell

**Comment by martinpl** [comment]
Maybe there will be some useful information / errors in:
1) ALT + F2 -> type lg -> Extensions -> Tray Icons Reloaded -> Display errors
or
2) In terminal `journalctl /usr/bin/gnome-shell -f -o cat` (will be bloated in unrelated stuff)

**Comment by aireilly** [comment]
Also seeing this error. F35, GNOME 41.1

Sys tray is functional with this extension: https://github.com/ubuntu/gnome-shell-extension-appindicator

**Comment by ryanpcmcquen** [comment]
Log out/in fixed it here. Fedora 35, Gnome 41.
